runners/mlbox_singularity_run/mlbox_singularity_run/__main__.py

```diff
@@ -0,0 +1,36 @@
+import os
+import typer
+from mlbox_singularity_run.singularity_run import SingularityRun
+from mlcommons_box import parse   # Do not remove (it registers schemas on import)
+from mlcommons_box.common import mlbox_metadata
+from mlbox_singularity_run import metadata
+
+
+metadata.MLSchemaRegistrar.register()
+app = typer.Typer()
```

Comment:
Same comment as above with respect to the usage Click over Typer. Again, sorry :(